tests/functional/object/mcg/test_block_external_access.py

```diff
@@ -0,0 +1,126 @@
+import logging
+import time
+
+import yaml
+
+from ocs_ci.framework.testlib import MCGTest, red_squad, mcg
+from ocs_ci.ocs import constants, ocp
+from ocs_ci.framework import config
+
+
+logger = logging.getLogger(__name__)
+
+
+def check_disable_routes(disable_routes_val):
+    """
+    Args:
+        disable_routes_val (bool) Value to be set to 'disableRoutes' parameter
```

Comment:
Please summarize what this function checks. It's confusing to read the asserts at the end without the needed context.

Reply:
Done. Thanks